website/content/en/preview/concepts/disruption.md

```diff
@@ -198,6 +197,39 @@ Karpenter enables this feature by watching an SQS queue which receives critical
 
 To enable interruption handling, configure the `--interruption-queue` CLI argument with the name of the interruption queue provisioned to handle interruption events.
 
+### Node Auto Repair 
```

Comment:
I wonder if we think about doing something like this where we document the feature state next to the feature description -- gives a quick overview that this isn't a stable feature and you could even include a pointer to the feature flag portion of the documentation

<img width="606" alt="Screenshot 2025-01-22 at 10 31 55 AM" src="https://github.com/user-attachments/assets/daaa74c6-43ff-483e-9ab2-ca16b95c6c2c" />